packages/notifications/src/elements/Alert.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { StyledAlert } from '../styled';
+import { VALIDATION } from '../utils/validation';
+
+export interface IAlertProps {
+  type?: VALIDATION;
+}
+
+/**
+ * Supports all `<div>` props
+ */
+export const Alert = StyledAlert as React.FC<IAlertProps>;
```

Comment:
This is downcasting from `forwardRefExoticComponent` to `functionComponent` and removes the ability to use the `ref` prop.

You can use `forwardRef` and use the `IStyledAlertProps` interface as the props.

Reply:
Should we also update `react-dropdowns` to not downcast to `React.FC`? I was following how [the Dropdown components](https://github.com/zendeskgarden/react-components/blob/next/packages/dropdowns/src/Fields/Message.ts#L19) were migrated to TypeScript.

Seems like a lot of our existing components don't support ref forwarding. Moving forward, I'll make sure to configure components to support ref forwarding if the component is exported to consumers.

Thanks for the suggestion, I'll go ahead and update these components and re-request a review.

Reply:
> Seems like a lot of our existing components don't support ref forwarding.

Yep, this wasn't very consistent before the CSS-in-JS migration. When it's time to move `react-dropdowns` over I will clean up those usages.